tests/test_relative_frame_task.py

```diff
@@ -142,7 +142,7 @@ def test_relative_jacobian(self):
         )
         self.configuration.update(q)
         relative_task.set_target_from_configuration(self.configuration)
-        J = relative_task.compute_jacobian(self.configuration)
+        J = -relative_task.compute_jacobian(self.configuration)
```

Comment:
I would add a comment to future travelers/self around here, e.g. "sign has flipped since we saved J_check here", but that's up to you :+1:

Reply:
Let us be even cleaner !